src/components/RegistrationList.tsx

```diff
@@ -159,6 +159,10 @@ export const RegistrationListItemContent = ({
         </TruncatableTypography>
       </ListItemText>
 
+      {location.pathname.includes(UrlPathTemplate.ResearchProfile) && isPromotedPublication && (
+        <StarIcon fontSize="small" />
+      )}
```

Comment:
Liten PS om at på mobilskjerm blir det mindre plass for "vanlig" innhold om det er en fremhevet publisering, og alt i "stjernas flexfelt" vil alltid være tomt. Ikke noe stort problem for min del, men greit å være bevisst på